src/test/resources/features/cs/ManagementUI.feature

```diff
@@ -454,9 +489,28 @@ Feature: ManagementUI
 
 #    MANAGE MPAM BUSINESS UNITS
 
+  @ListsManagement
+  Scenario: User is able to create new business units through Lists management
+    When I select to "Manage MPAM Business Units"
+    And I load the business units for the "UKVI" business area
+    And I create a new business unit
+    Then a success message is displayed
+    And I select to "Manage MPAM Business Units"
```

Comment:
I would like to see us store the selected business area as a session variable and then move both these steps into the glue for the bottom assertion.